modules/lightning_telemetry/src/Telemetry.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace Drupal\lightning_telemetry;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Extension\ModuleExtensionList;
+use GuzzleHttp\ClientInterface;
+
+/**
+ * Telemetry service.
+ */
+class Telemetry {
+
+  /**
+   * Amplitude API URL.
+   */
+  const AMPLITUDE_API_URL = 'https://api.amplitude.com/httpapi';
```

Comment:
These constants should probably be configuration values.

Reply:
They should never change though. Why not make them constants?

Reply:
I agree that unless there are different endpoints (like dev > stage > prod, or something) there's no reason to make this configurable.